GETTING_STARTED.md

```diff
@@ -0,0 +1,106 @@
+# Getting started
+
+Firstly, start the application by running `python3 main.py`.
+
+You will be shown the main window of the application (pictured below) which contains both a list of components and a 3D view visualising the components listed with their respective shape and position. 
+The sample component is added by default, and is indicated by the red cube. You can move around the 3D view by using the arrow keys to move, click+drag to pan, and pgup/down to zoom.
+
+![](resources/images/NeXusConstructor_001.png)
+
+By clicking the "Nexus File Layout" tab in the left pane of the window, the NeXus file layout can be shown in a tree structure.
```

Comment:
Try to use the text to explain what and who the feature is for rather than describe what can be seen anyway in the screenshot.
"If you are familiar with the NeXus format it may be useful to view the layout of the resulting NeXus file as you modify the template by adding components and other information. You can do this by clicking the "Nexus File Layout" tab in the left pane of the main window."